published/088-2020-may-5.md

```diff
@@ -0,0 +1,86 @@
+* [Alex Potsides on Twitter: “js-IPFS will be getting the same improvements to bitswap (e.g. transfer speeds) very soon!   Take a look at the roadmap here: https://t.co/OAhLUkKx5P… https://t.co/mEbkhnfMaJ”](https://twitter.com/achingbrain/status/1255562111436103680?s=20)
+* https://medium.com/equilibriumco/rust-ipfs-append-only-log-001-42106983f3c
+
+## Video
+https://www.youtube.com/watch?v=G8FvB_0HlCE&feature=emb_logo
+
+## 0.5 Release news
+* [Decentralized Web Protocol IPFS Has Its Biggest Update So Far](https://cointelegraph.com/news/decentralized-web-protocol-ipfs-has-its-biggest-update-so-far)
+* [The InterPlanetary File System gets a network upgrade to make data fetching faster - The Block](https://www.theblockcrypto.com/post/63189/ipfs-network-upgrade-data-fetching)
+* [Latest Version of Open Source IPFS Improves Performance - DevOps.com](https://devops.com/latest-version-of-open-source-ipfs-improves-performance/)
+* [IPFS Releases 0.5.0 Network Upgrade, Finding and Fetching Data Is Now 5 Times Faster](https://bitcoinexchangeguide.com/ipfs-releases-0-5-0-network-upgrade-finding-and-fetching-data-is-now-5-times-faster/)
+* [Ya fue liberada la nueva versión del sistema de archivos descentralizado IPFS 0.5](https://blog.desdelinux.net/ya-fue-liberada-la-nueva-version-del-sistema-de-archivos-descentralizado-ipfs-0-5/)
+* [Protocolo IPFS é atualizado para tornar coleta de dados mais rápida – Money Times](https://www.moneytimes.com.br/protocolo-ipfs-e-atualizado-para-tornar-coleta-de-dados-mais-rapida/)
+
+
+## Top highlights this week
+* [IPFS and the Future of the Internet - DLT Labs - Medium](https://medium.com/@dltlabs/ipfs-and-the-future-of-the-internet-dfb3cfed0531)
+* 3 Part Tutorial: [Nim-libp2p Tutorial: A Peer-to-Peer Chat Example (1)](https://our.status.im/nim-libp2p-tutorial-a-peer-to-peer-chat-example-1/)
+* [Algorand Developer Portal](https://developer.algorand.org/solutions/creating-point-sale-application-algorand-blockchain/)
+* How to Save Time on DevOps and Scale With Infura: Lessons from Leading Ethereum Dapps https://consensys.net/blog/blockchain-development/how-to-save-time-on-devops-and-scale-with-infura-lessons-from-leading-ethereum-dapps/
+* https://nos.chat/t/update-nos-core-2-8-0-decentralized-validator-profiles/185
+
+
+## People are building the coolest stuff with IPFS
+* [Best YouTube Alternatives 2020: Decentralized Video Platforms](https://blockonomi.com/youtube-alternative/)
+* https://medium.com/qri-io/how-to-use-qri-features-to-build-a-better-dataset-2d236da626f0
+* [Release Updates - ENS Domains on Your IPFS Site! - Fleek Blog](https://blog.fleek.co/posts/Fleek-Release-ENS-Domains)
+* [Switcheo Exchange Hosted on IPFS](https://blog.switcheo.network/switcheo-exchange-hosted-on-ipfs/)
+* [Ocean Protocol 💖 IPFS](https://ipfs.oceanprotocol.com/)
+* https://simpleaswater.com/intro-to-aviondb/
```

Comment:
[AvionDB: A MongoDB-like, Distributed Database](https://github.com/dappkit/aviondb)

It would be preferable to add the Github link (https://github.com/dappkit/aviondb) to the project. In case we can add both blog+github links, that would be even better.